cfgov/unprocessed/js/apps/financial-well-being/fwb-questions.js

```diff
@@ -0,0 +1,83 @@
+/* eslint no-global-assign: "off" */
+'use strict';
+
+var Analytics = require( '../../modules/Analytics' );
+
+function init() {
+
+  var totalQuestions = 12;
```

Comment:
Could be styled as a constant `const TOTAL_QUESTIONS = 12`.

Reply:
Updated in the next PR.